packages/teleterm/package.json

```diff
@@ -17,7 +17,8 @@
     "build-main": "webpack build --config webpack.main.config.js --progress --mode=production",
     "build-renderer": "webpack build --config webpack.renderer.prod.config.js --progress",
     "build-natives": "electron-builder install-app-deps",
-    "package": "electron-builder build --publish never -c.extraMetadata.name=teleconnect"
+    "package": "electron-builder build --publish never -c.extraMetadata.name=teleconnect",
+    "generate-grpc-shared": "npx -y --target_arch=x64 --package=grpc_tools_node_protoc_ts --package=grpc-tools -- grpc_tools_node_protoc -I=src/sharedProcess/api/proto --ts_out=service=grpc-node,mode=grpc-js:src/sharedProcess/api/protogen --grpc_out=grpc_js:src/sharedProcess/api/protogen --js_out=import_style=commonjs,binary:src/sharedProcess/api/protogen src/sharedProcess/api/proto/*.proto"
```

Comment:
> Unfortunately, we still can't use `grpc-tools` on M1 Macs, so I use `npx` to install a version for x64 arch.

I think there might be an even easier way of dealing with this where we can just tell yarn/npm that we always want the x64. I'll check it out later, I vaguely remember something like that existing.

Reply:
I tried to search for that, but I think I meant the `target_arch` flag which you're already using here and which cannot be specified in package.json.

We could probably use one of the package-specific node-pre-gyp flags or whatever, but idk how to do this and it's probably not worth the time anyway since this solution here is good enough.

Could we lock the package versions though? I think it should be a matter of appending `@x.y.z` to package names in the `npx` command.

Reply:
Yes, I think it is not possible to install all packages for the target arch.  
Added exact versions to the `npx` script.